sklearn/iris_sklearn_trainer/iris.py

```diff
@@ -10,11 +10,29 @@
 
 from sklearn import svm
 from sklearn.externals import joblib
+from sklearn.model_selection import train_test_split
 
-# Fill in your Cloud Storage bucket name
-BUCKET_ID = <YOUR_BUCKET_NAME>
-# [END setup]
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    '--job-dir',
+    help='GCS location to write checkpoints and export models',
+    required=True
+)
+parser.add_argument(
+    '--kernel',
+    help='Specifies the kernel type to be used in the '
+         'algorithm.It must be one of `linear`, `poly`, '
+         '`rbf`, `sigmoid` or a callable',
+    default='rbf'
+)
+parser.add_argument(
+    '--c',
+    help='Penalty parameter C of the error term',
+    type=float,
+    default='1.0'
+)
 
+args = parser.parse_args()
```

Comment:
parsing should be inside a `if __name__ == '__main__':` statement at the bottom of the python file.